linux_os/guide/services/sssd/sssd_enable_certmap/oval/shared.xml

```diff
@@ -0,0 +1,21 @@
+<def-group>
+      <definition class="compliance" id="{{{ rule_id }}}"  version="1">
+          {{{ oval_metadata("SSD should be configured to authenticated identity to the user or group account
+             by using certmap.") }}}
+          <criteria>
+              <criterion comment="check if certmap is enabled in /etc/sssd/sssd.conf"
+                         test_ref="test_{{{rule_id}}}_section" />
+         </criteria>
+      </definition>
+    <ind:textfilecontent54_test check="all" check_existence="all_exist"
+                                comment="check if certmap is enabled in /etc/sssd/sssd.conf"
+                                id="test_{{{rule_id}}}_section" version="1">
+        <ind:object object_ref="obj_{{{rule_id}}}_section" />
+    </ind:textfilecontent54_test>
+
+     <ind:textfilecontent54_object id="obj_{{{rule_id}}}_section" version="1">
+         <ind:filepath>/etc/sssd/sssd.conf</ind:filepath>
+         <ind:pattern operation="pattern match">^\[certmap\/.+\/.+\]$</ind:pattern>
+         <ind:instance datatype="int" operation="equals">1</ind:instance>
```

Comment:
I think you should use template lineinfile instead. It;s possible to generate only OVAL and turn off the Ansible and Bash:
```
template:
    name: lineinfile
    vars:
      path: '/etc/sssd/sssd.conf'
      text: '^\[certmap\/.+\/.+\]$'
    backends:
        ansible: "off"
        bash: "off"
```